app/controllers/mobility_devices_controller.rb

```diff
@@ -48,8 +48,7 @@ def destroy
   private
 
   def set_device
-    @device = MobilityDevice.find_by id: params.require(:id)
-    head :not_found if @device.blank?
+    @device = MobilityDevice.find params.require(:id)
```

Comment:
```suggestion
    @device = MobilityDevice.find params[:id]
```
`.find` calls generally don't need `params.require`- it's okay if it comes in blank because `.find` will throw a `RecordNotFound` anyways